### ISSUE TITLE
Unable to change lex-web-ui.js

### ISSUE THREAD

**Issue by Ramozz** [aws-samples/aws-lex-web-ui#340]
Good afternoon,
I tryied to change the widget source code, in order to include additional buttons on the chatbot UI, my two main goals were:
1. Introduce a refresh button to clear all the application information inside the chatbot UI
2. Trigger a different disclaimer, when a different language is selected

I manage to make the changes locally, by changing the ToolbatContainer.Vue and other Vue components and everything works fine, locally. However when I try to change my s3 bucket source code, in order to use the codebuild to minify my files and generate the scripts for the s3 webapp bucket, based on the source code used locally, I'm never able to generate the min.js, js.map and min.css from the lex-web-ui.js so the changes in the widget used by the main website are never reflected, even though that locally everything is working fine. 
I have done already npm run build in the correct directory, like running the comands specified in the CodeBuild project, but as said before the files related with lex-web-ui.js are never created again. 
To summarize, I would like to ask if someone could help me to understand, how can I generate the production files and even the lex-web-ui.js which looks like a generated file, in order to pass the changes in the local UI to the website deployed in AWS, however as I said before I'm not able to generate those files, if someone could help it would be amazing, have a nice day!

**Comment by bobpskier** [comment]
Congrats on getting the modifications to ToolbarContainer.Vue and related components setup. The way you can perform a full build is using the build/release.sh script. The end result will be a set of files in the dist folder. 

Next typically I'll setup an S3 bucket as a bootstrap bucket where I can install using cloud formation install or update via template in this S3 bucket. 

The full process would be:

```
cd build
./release.sh
export BUCKET=nameOfMyBootstrap 
./upload-bootstrap.sh
```

You can then use the CloudFormation console to create a new stack and point at your template in your bootstrap bucket. The only requirements when using your own bootstrap bucket is to allow objects in the bootstrap bucket to be public. The bucket itself is not a public bucket. Just objects in the bucket can be made public. Next, when using the template make sure to change the bootstrap bucket template parameter to use the name of your bootstrap bucket and change the artifacts parameter to be just "artifacts" which is where upload-bootstrap.sh will place items used during the install.

You can either use the files directly in /dist or setup a bootstrap bucket if you are installing multiple instances of lexwebui.

**Comment by bobpskier** [comment]
Added new section in top level readme on use of release and bootstrap bucket.
